lib/extensions/ar_migration.rb

```diff
@@ -82,7 +95,7 @@ def restart_subscription
 
 module ArPglogicalMigration
   def migrate(direction)
-    PglogicalSubscription.all.each do |s|
+    ArPglogicalMigrationHelper.subscriptions.each do |s|
       ArPglogicalMigrationHelper::RemoteRegionMigrationWatcher.new(s, version.to_s).wait_for_remote_region_migration
```

Comment:
Not for this PR, but I was thinking about this part, and since the RemoteRegionWatcher is in the helper anyway, there's no point in having the `ArPglogicalMigrationHelper.subscriptions` call out here at all, cause it's all inside baseball (or even as a public method on the module)...I'm not even sure why we should expose any of that outside of the helper...I'm picturing a refactoring like:

```ruby
module ArPglogicalMigrationHelper
  def self.around_migration(version, direction)
    subscriptions.each do |s|
      RemoteRegionMigrationWatcher.new(s, version.to_s).wait_for_remote_region_migration
    end

    yield.tap do
      update_local_migrations_ran(version.to_s, direction)
    end
  end

  # ...  and all of the other methods are private
end

# Then here do:

def migrate(direction)
  ArPglogicalMigrationHelper.around_migration { super }
end
```

You could even extract the before_migration and after_migration bits to separate methods to make it even more readable, so it would look like:

```ruby
module ArPglogicalMigrationHelper
  def self.around_migration(version, direction)
    before_migration(version)
    yield.tap do
      after_migration(version, direction)
    end
  end
```